cdap-app-fabric/src/main/java/io/cdap/cdap/internal/credential/store/CredentialProfileStore.java

```diff
@@ -38,14 +42,26 @@
 import java.util.Spliterators;
 import java.util.stream.Collectors;
 import java.util.stream.StreamSupport;
+import javax.inject.Inject;
+import javax.inject.Named;
 
 /**
  * Storage for credential identities.
  */
 public class CredentialProfileStore {
 
+  private static final byte[] CREDENTIAL_PROFILE_STORE_AD = CredentialProviderStore.class
+      .getCanonicalName().getBytes(StandardCharsets.UTF_8);
   private static final Gson GSON = new Gson();
 
+  private final AeadCipher dataStorageCipher;
+
+  @Inject
+  public CredentialProfileStore(@Named(DataStorageAeadEncryptionModule.DATA_STORAGE_ENCRYPTION)
```

Comment:
injected constructors should generally be package private to prevent manual construction

Reply:
It was originally needed since the stores were in their own directory, but I moved it to the same directory as test classes since that seems unnecessary. Done